app/helpers/skin_cache_helper.rb

```diff
@@ -1,6 +1,6 @@
 module SkinCacheHelper
   def cache_timestamp
-    Time.now.utc.to_s(:usec)
+    Time.now.utc.to_fs(:usec)
```

Comment:
I saw a deprecation warning for this during Cucumber tests, so I figured I'd change it to avoid pain later. (Similar change below.)